docs/software/uenv/using.md

```diff
@@ -0,0 +1,445 @@
+
+??? warning "Failed to unshare the mount namespace"
+
+    If you get the following error message when starting a uenv:
+    ```console
+    $ uenv start linalg/24.11:v1
+    squashfs-mount: Failed to unshare the mount namespace: Operation not permitted
+    ```
+    you most likely already have a uenv mounted.
+    The `uenv status` command will report that you have a uenv loaded if that is the case:
+    ```console
+    $ uenv status
+    prgenv-gnu:/user-environment
+      GNU Compiler toolchain with cray-mpich, Python, CMake and other development tools.
+      views:
+        spack: configure spack upstream
+        modules: activate modules
+        default:
+    ```
+    Unload the active uenv by exiting the current shell before loading the new uenv.
+
+The basic syntax of uenv start is `uenv start image` where `image` is the uenv to start.
+The image can be a [label][ref-uenv-labels], the hash/id of the uenv, or a file:
+
+!!! example "uenv start"
+    ```console
+    # start the image using the name of the uenv
+    $ uenv start netcdf-tools/2024:v1
+
+    # or use the unique id of the uenv
+    $ uenv start 499c886f2947538e
+
+    # or provide the path to a squashfs file
+    $ uenv start $SCRATCH/my-uenv/gromacs.squashfs
+    ```
+
+!!! warning "do not use `uenv start` in scripts or `~/.bashrc`"
+    The `uenv start` command is only for creating interactive environments, because it creates an interactive shell.
+    For Slurm jobs, and use inside scripts, use the `uenv run` and Slurm integration.
+
+    Because uenv start and run execute commands in a new environment, they [can't be used in bashrc][ref-guides-terminal-bashrc] to configure your environment during login.
+    See our guide for creating [convenient custom environments][ref-uenv-customenv] with uenv for alternatives to `module load` in your `~/.basrhc`.
+    
+
+[](){#ref-uenv-run}
+## Running a uenv
+
+The `uenv run` command can be used to run an application or script in a uenv environment, and return control to the calling shell when the command has finished running.
+
+The run command is very useful when scripting complicated workflows, and can be used to create customized environments.
+See the guide to [creating custom environments][ref-uenv-customenv] for an example.
+
+!!! info "how is `uenv run` different from `uenv start`?"
+    `uenv start` sets up the uenv environment, then starts an interactive shell in that environment.
+    When you are finished, you can type `exit` to finish the session.
+
+    `uenv run` is more generic - instead of running a shell in environment, it takes the executable and arguments to run in the shell.
+    The following commands are equivalent:
+
+    ```console
+    # start a new bash shell in prgenv-gnu
+    uenv start prgenv-gnu/24.11
+    # start a new bash shell in prgenv-gnu
+    uenv run prgenv-gnu/24.11 -- bash
+    ```
+
+!!! example "running cmake"
+    Call `cmake` to configure a build with the `default` view loaded
```

Comment:
Just to make it clear the value here is using the cmake inside the uenv (e.g. because it is a newer version)

```suggestion
    Call the `cmake` provided by the uenv to configure a build with the `default` view loaded
```